gustaf/spline/ffd.py

```diff
@@ -233,29 +224,32 @@ def _deform(self):
                     "spline or(and) the mesh are not yet defined. "
                     "Please set either spline or mesh."
             )
-        if self._is_calculated and self._is_calculated_trackable:
-            return None
+        if self._spline._data.get("gustaf_ffd_computed", False):
+           return 
+
+        spline = self._spline.copy()
+        if "knot_vectors" in spline.required_properties:
```

Comment:
```suggestion
        if spline.has_knot_vectors:
```
let's promote new property